docs/aerodromes/Hobart.md

```diff
@@ -35,44 +35,42 @@ Refer to [Class D Tower Skills](../../controller-skills/classdtwr/) for more inf
 
 Jet Aircraft planned via **CLARK**, **LATUM**, or **LAVOP**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
 
-Jet Aircraft **not** planned via **CLARK**, **LATUM**, or **LAVOP**, shall be assigned an appropriate **Procedural SID** or a visual departure.
-
 Non-Jet Aircraft planned via **CLARK**, **KANLI**, or **LAVOP**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
 
-Non-Jet Aircraft **not** planned via **CLARK**, **KANLI**, or **LAVOP**, shall be assigned an appropriate **Procedural SID** or a visual departure.
+Other aircraft shall be assigned an appropriate **Procedural SID** or a visual departure.
 
 ## Cambridge (YCBG)
 
 HB ADC and/or HB SMC is resposible for clearances into and out of Cambridge (YCBG).
 
-Prior to leaving APN, aircraft will establish communications with HB SMC for Airways Clearance.
+Prior to leaving the apron, aircraft will establish communications with HB SMC for Airways Clearance.
 
 !!! Example
-    **NDR** -> **HB SMC**: "Hobart Ground, NDR, for Devonport, Request Clearance"  
-    **HB ADC** -> **NDR**: "NDR, Cleared to Devonport via KANLI flight plan route, KANLI# Departure, Climb via SID A045, Squawk 4432"
+    **NDR:** "Hobart Ground, NDR, for Devonport, Request Clearance"  
+    **HB ADC:** "NDR, Cleared to Devonport via KANLI flight plan route, KANLI3 Departure, Climb via SID A045, Squawk 4432"
```

Comment:
should this be SMC?

Reply:
Good pickup, ADC is probably more appropriate

Reply:
Actually, from reading the ERSA, looks like SMC for airways clearance, ADC before leaving the apron for traffic